src/nav/NavButtonGeneral.js

```diff
@@ -26,7 +26,14 @@ export default class NavButtonGeneral extends PureComponent<Props> {
 
     return (
       <Touchable onPress={onPress}>
-        <View style={this.styles.navButtonFrame}>{children}</View>
+        <View
+          accessible={true}
+          accessibilityLabel="Go Back!"
```

Comment:
This is much too broad, because this component is used in a number of other places in the app.

For a clean approach to apply this change in only the relevant spots, see these commits in the earlier PR:
https://github.com/zulip/zulip-mobile/pull/2753/commits/6935e5bfb54b1279805228f32737c8d51b5e5686
https://github.com/zulip/zulip-mobile/pull/2753/commits/d26962d0116927e4bdeb8e3d810ec1a4645b6bb6
https://github.com/zulip/zulip-mobile/pull/2753/commits/a0d74d9b8d92d2bab226d45b598267bba8c4d05d